src/main/java/cz/cvut/kbss/termit/rest/TermController.java

```diff
@@ -722,9 +733,21 @@ public List<AbstractChangeRecord> getHistory(@Parameter(description = ApiDoc.ID_
                                                  @PathVariable String localName,
                                                  @Parameter(description = ApiDoc.ID_STANDALONE_NAMESPACE_DESCRIPTION,
                                                             example = ApiDoc.ID_STANDALONE_NAMESPACE_EXAMPLE)
-                                                 @RequestParam(name = QueryParams.NAMESPACE) String namespace) {
+                                                 @RequestParam(name = QueryParams.NAMESPACE) String namespace,
+                                                 @Parameter(description = "Change type used for filtering.")
+                                                 @RequestParam(name = "type", required = false) URI changeType,
+                                                 @Parameter(description = "Author name used for filtering.")
+                                                 @RequestParam(name = "author", required = false,
+                                                               defaultValue = "") String authorName,
+                                                 @Parameter(description = "Changed attribute name used for filtering.")
+                                                 @RequestParam(name = "attribute", required = false,
+                                                               defaultValue = "") String changedAttributeName) {
         final URI termUri = idResolver.resolveIdentifier(namespace, localName);
-        return termService.getChanges(termService.findRequired(termUri));
+        final ChangeRecordFilterDto filter = new ChangeRecordFilterDto();
```

Comment:
I would add a constructor taking all four fields as parameters to `ChangeRecordFilterDto`, since we are setting them here all anyway.